packages/openapi-v3/src/request-body-decorator.ts

```diff
@@ -0,0 +1,148 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  RequestBodyObject,
+  SchemaObject,
+  ReferenceObject,
+} from '@loopback/openapi-v3-types';
+import {MetadataInspector, ParameterDecoratorFactory} from '@loopback/context';
+import {getSchemaForRequestBody} from './';
```

Comment:
import from the file itself please

Reply:
Done 👍 
I delete exporting `generate-schema.ts` in `index.ts`, the functions in it are only for internal use